lib/active_admin/reform/resource_controller/data_access.rb

```diff
@@ -0,0 +1,37 @@
+require 'reform'
+require 'active_support/concern'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.